lib/pages/character/inventory.dart

```diff
@@ -90,61 +90,77 @@ class InventoryPage extends StatelessWidget {
   Widget _buildBag(BuildContext context, Bags bag, int bagIndex) {
     List<InventoryItem> inventory = bag.inventory.where((i) => i.id != -1 && i.itemInfo != null).toList();
     int usedSlots = inventory.length;
+    FontStyle fontStyle = (usedSlots == bag.size ? FontStyle.italic : FontStyle.normal);
     return CompanionCard(
+      padding: EdgeInsets.all(0.0),
```

Comment:
This can be changed to `EdgeInsets.zero` which looks a bit neater.

Reply:
I've done this.